### PR TITLE
added hardware ref, mirrored software ref. Also tweaked the loot drop…

### LINKED ISSUE #18: Update data structure for Hardware / Software items

### PR DESCRIPTION
**Pull request by Thaewyn**
resolves #18

### CODE REVIEW

File: db/software_ref.json [hunk 1]
```diff
@@ -44,6 +47,7 @@
     "power": 5,
```
Comment (Thaewyn) [inline db/software_ref.json:48]: Test comment. Add item urls

Comment (Thaewyn) [reply db/software_ref.json:48]: Non-issue, fixing elsewhere

Comment (phopps) [reply db/software_ref.json:48]: Test reply

**Review by phopps — APPROVED** [review]
LGTM, good catch with item urls

**Pull request merged**